resources/aws/elb.go

```diff
@@ -225,7 +225,7 @@ func (lb ELB) findExisting() (*elb.LoadBalancerDescription, error) {
 	descriptions := resp.LoadBalancerDescriptions
 
 	if len(descriptions) == 0 {
-		return nil, NamedResourceNotFoundError{Name: lb.Name}
+		return nil, NotFoundError{ResourceType: ELBType, Name: lb.Name}
```

Comment:
My personal preference would be to keep the error handling consistent. I only see that this is about the error message. We can have one constant for the error message format if this is the problem. At the end, do what you think is right.